_data/providers.json

```diff
@@ -1,6 +1,6 @@
 {
   "description_templates": {
-    "default": "Get Geyser as a plugin, as in the [setup guide](https://wiki.geysermc.org/geyser/setup/). Use the same port as your Java server for the Bedrock port in your config (either by setting it yourself, or enabling \"clone-remote-port\") and connect with the same IP and port as you would on Java."
+    "default": "Get Geyser as a plugin, as in the [setup guide](/geyser/setup/). Use the same port as your Java server for the Bedrock port in your config (either by setting it yourself, or enabling \"clone-remote-port\") and connect with the same IP and port as you would on Java."
```

Comment:
Please revert this, this is done on purpose. For example. the Geyser discord bot depends on this file to show up to date provider instructions, so any relative link wouldn't work.

Also, there's another PR on the wiki that's changing the providers.json quite drastically, so I would avoid changes to this file that go beyond updating links/fixing typos